4. User Guides/README.md

```diff
@@ -59,7 +67,7 @@ Find existing channels by clicking on the <img src="More channels.png" alt="More
 
 There are two notions of leaving a channel:
 
-1. To **hide** a channel is to remain a member of the channel but to remove it from your list of channels on the left-hand pane. 
+1. To **hide** a channel is to remain a member of the channel but to remove it from your list of channels on the left-hand pane.
 1. To **leave** a channel is to remove it from your list of channels _and to leave the channel_ (i.e., no longer be a member of the channel).
 
 Every channel has an **owner**, and the owner of a channel cannot leave a channel until setting someone else as the owner of that channel. You can do it via right side panel (Flex tab) and "Member list".
```

Comment:
Ok noted, will mention this in the "Hide Channel" article. This line, like others in the file, only had white space removed from the end and did not have any content changed.